api_auth.gemspec

```diff
@@ -9,18 +9,20 @@ Gem::Specification.new do |s|
   s.authors = ['Mauricio Gomes']
   s.email = 'mauricio@edge14.com'
 
-  s.add_development_dependency 'appraisal'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'amatch'
-  s.add_development_dependency 'rspec', '~> 3.4'
+  s.required_ruby_version = '>= 2.1.0'
```

Comment:
Can this be >= 2.0, or does anything here require 2.1?

Reply:
The support of ruby 2.0 has ended the february 2016.
https://www.ruby-lang.org/en/news/2016/02/24/support-plan-of-ruby-2-0-0-and-2-1/

The last version of rubocop drop the support of ruby 2.0
https://github.com/bbatsov/rubocop/commit/753227914c1ad311e6116311fc1ef76d9b0904dc#diff-458bc38d82061cdac508dfb1be2b918b

So if we continue to use rubocop, we have the implicit configuration `TargetRubyVersion: 2.1`. We can not be sure that we don't break the Ruby 2.0 synthax.

Reply:
Moreover, we don't test actually 2.0 with travis-ci!

Reply:
Ah yeah, good point.